h/session.py

```diff
@@ -55,9 +59,13 @@ def _current_groups(request):
 
 
 def includeme(config):
-    config.add_settings({
-        "redis.sessions.cookie_httponly": True,
-        "redis.sessions.cookie_max_age": 2592000,
-        "redis.sessions.timeout": 604800,
-    })
-    config.include('pyramid_redis_sessions')
+    settings = config.registry.settings
+
+    factory = SignedCookieSessionFactory(
+        secret=derive_key(settings['secret_key'], b'h.session.cookie_secret'),
+        hashalg='sha512',
+        max_age=2592000,
```

Comment:
The lovely thing about getting rid of this is that we can now *drastically* reduce the session and cookie lifetimes, because it doesn't have anything to do with login.

I'd suggest removing `max_age`, and allowing session cookies to be browser-scoped, and reducing `timeout` to, say, an hour?